Kitodo/src/main/java/org/kitodo/exporter/ExportXmlLog.java

```diff
@@ -150,20 +148,11 @@ public void startExport(Iterable<Process> processList, OutputStream outputStream
         outp.setFormat(Format.getPrettyFormat());
 
         try {
-
             outp.output(answer, outputStream);
+            outputStream.close();
```

Comment:
Are you sure you can move it here? If the exception is thrown in the line above, the stream is not closed.